src/main/java/com/northconcepts/datapipeline/examples/csv/ReadAndWriteCSVAsString.java

```diff
@@ -0,0 +1,17 @@
+package com.northconcepts.datapipeline.examples.csv;
+
+import com.northconcepts.datapipeline.csv.CommaSeparatedValues;
+
+public class ReadAndWriteCSVAsString {
+
+    private static final String[] CONTINENTS = {"Africa", "Antarctica", "Asia", "Australia", "Europe", "North America", "South America"};
+
+    public static void main(String[] args) {
```

Comment:
This is a good.  Let's also use `CommaSeparatedValues.fromValue(String)` to show reading a single String containing CSVs.